lib/sen_tweet_web/templates/page/index.html.eex

```diff
@@ -1,5 +1,19 @@
-<div id="messages-container">
+<nav class="level">
+  <div class="level-item has-text-centered">
```

Comment:
More metrics can be added to the UI here by copying this `<div` and its contents.